docs/supported_apis/dataframe_supported.rst

```diff
@@ -1,676 +1,45 @@
 ``pd.DataFrame`` supported APIs
 ===================================
 
-The following table lists both implemented and not implemented methods. If you have need
-of an operation that is listed as not implemented, feel free to open an issue on the
-`GitHub repository`_, or give a thumbs up to already created issues. Contributions are
-also welcome!
+The ``pd.DataFrame`` supported APIs table lists both implemented and not implemented methods.
+If you have need of an operation that is listed as not implemented, feel free to open an
+issue on the `GitHub repository`_, or give a thumbs up to already created issues. Contributions
+are also welcome!
 
-The following table is structured as follows: The first column contains the method name.
-The second column contains link to a description of corresponding pandas method.
-The third column is a flag for whether or not there is an implementation in Modin for
-the method in the left column. ``Y`` stands for yes, ``N`` stands for no, ``P`` stands
-for partial (meaning some parameters may not be supported yet), and ``D`` stands for
-default to pandas.
+Supported APIs table is structured as follows: The first column contains the method name,
```

Comment:
These tables are structured in bit other way ([link](https://github.com/modin-project/modin/pull/4402/files#diff-506c226c0b2753ce3bbf33ea6c4de22796376d982b483a4d9bbbb3e3e6561a9dR17)). Am I correct?

Reply:
We have a table for all methods and a table for a particular method. We should reflect this.

Reply:
Yes, this part looks outdated. Fixed.